src/main/java/com/siatsenko/movieland/web/dto/MovieDetailDto.java

```diff
@@ -28,7 +30,7 @@ public MovieDetailDto(Movie movie) {
         this.yearOfRelease = movie.getYearOfRelease();
         this.description = movie.getDescription();
         this.rating = movie.getRating();
-        this.price = movie.getPrice();
+        this.price = new BigDecimal(movie.getPrice()).setScale(2, RoundingMode.UP).doubleValue();
```

Comment:
create class DtoConverter, and put all convert methods there. 
Try to use ModelMapper